### PR TITLE
Sonja/update mod dash 2

### PR DESCRIPTION
**Pull request by Sonjajo**
Updated the moderator dashboard to have scrolling and added a few buttons. Redid the layout for the orgDetail that pops up to match figma. 
Still to be done:
- fix styles of left AppBar with the treasury version that Elizabeth and I talked about: https://mui-treasury.com/styles/tabs/#Apple
- add drawer for Notepad button
- add popup window for rejection history
- look into custom drawer left scrollbar, material UI doesn't have functionality we need

_Required for visual changes_

image

CC: @kwfk

### CODE REVIEW

**Comment by elixawu** [comment]
minor change, but the accept and decline buttons on the bottom should be swapped order (accept on right)

File: pages/mod2.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import InboxIcon from '@material-ui/icons/MoveToInbox';
+import MailIcon from '@material-ui/icons/Mail';
```
Comment (kwfk) [inline pages/mod2.tsx]: collapse to one line import
i.e.
```typescript
import { Drawer, List } from '@material-ui/core'
```

File: pages/mod2.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+} from '@material-ui/core';
+import clsx from 'clsx';
+import {
+  makeStyles,
+  useTheme,
+  Theme,
+  createStyles,
+} from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import CssBaseline from '@material-ui/core/CssBaseline';
```
Comment (kwfk) [inline pages/mod2.tsx]: this is imported at base root so don't need to be imported here

File: styles/Moderator.module.css [hunk 3]
```diff
@@ -1,47 +1,60 @@
...
-  margin-left: 15px;
-  margin-right: 15px;
+  width: 400px;
+  height: 100%;
+  padding-left: 40px;
+  padding-right: 20px;
 }
+
 .rightCol {
+  width: 70%;
```
Comment (kwfk) [inline styles/Moderator.module.css]: if using a percentage, you want the elements on the same level to equal 100%

File: styles/Moderator.module.css [hunk 4]
```diff
@@ -1,47 +1,60 @@
...
 }
 .leftCol {
   display: flex;
   justify-content: flex-start;
   flex-direction: column;
-  width: 25%;
-  margin-top: 8px;
-  margin-left: 15px;
-  margin-right: 15px;
+  width: 400px;
```
Comment (kwfk) [inline styles/Moderator.module.css]: if using pixels, you want the other element in the same level to span the rest of the width

File: pages/registration.tsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import { useFormik } from 'formik';
-import { Form } from 'interfaces';
+import { Form } from 'interfaces/index';
```
Comment (kwfk) [inline pages/registration.tsx]: was this change necessary?

File: components/moderator/OrgDetail/index.tsx [hunk 6]
```diff
@@ -1,42 +1,117 @@
...
       <div className={styles.row}>
-        <div className={styles.response}>
-          Contact Name <br />
-          <div className={styles.infoBox}>{items.contact}</div>
+        <div className={styles.section}>
+          <div className={styles.small}>Orientation</div>
+          <div className={styles.chips}>
+            {items.orientation.map((item) => (
+              // TODO: Add accessibility support
+              // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
```
Comment (kwfk) [inline components/moderator/OrgDetail/index.tsx]: these eslint-disables are actually not needed!

File: styles/Moderator.module.css (hunk 4 above)
Comment (kwfk) [reply styles/Moderator.module.css]: i think leaving this as a pixel width would be better

File: styles/Moderator.module.css (hunk 3 above)
Comment (kwfk) [reply styles/Moderator.module.css]: then changing this as `calc(100% - 400px)`

**Review by kwfk — CHANGES_REQUESTED** [review]
nice work! it looks really nice. there are some minor changes i commented

you may want to add spacing between the demographic chips

File: pages/registration.tsx (hunk 5 above)
Comment (Sonjajo) [reply pages/registration.tsx]: nope you're right, i reverted it

File: interfaces/index.ts [hunk 7]
```diff
@@ -35,6 +35,27 @@ export type Form = {
...
+  missionStatement: string;
+  shortHistory: string;
+  keyValue: string;
+  lgbtqDemographic: string;
+  raceDemographic: string;
+  ageDemographic: string;
+  capacity: number;
+  ein: number;
+  foundingDate: number;
+};
```
Comment (kwfk) [inline interfaces/index.ts]: is this being used?

File: pages/moderator.tsx [hunk 8]
```diff
@@ -1,57 +1,205 @@
...
     </Layout>
   );
 };
 
-export const getStaticProps: GetStaticProps = async () => {
-  const items: OrgApp[] = sampleOrgAppData;
+// export const getStaticProps: GetStaticProps = async () => {
+//   const items: OrgApp[] = sampleOrgAppData;
+//   return { props: { items } };
+// };
```
Comment (kwfk) [inline pages/moderator.tsx]: let's remove this

File: pages/moderator.tsx [hunk 9]
```diff
@@ -1,57 +1,205 @@
...
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```
Comment (kwfk) [inline pages/moderator.tsx]: remove unused prop `context`

File: pages/moderator.tsx [hunk 10]
```diff
@@ -1,57 +1,205 @@
...
+
+  const handleDrawerOpenRight = (): void => {
+    setOpenRight(true);
+  };
+
+  const handleDrawerCloseRight = (): void => {
+    setOpenRight(false);
+  };
+
+  console.log(items);
```
Comment (kwfk) [inline pages/moderator.tsx]: remove console log

**Review by kwfk — CHANGES_REQUESTED** [review]
looking good! just some minor clean up

**Review by kwfk — APPROVED** [review]
Nice work!

**Pull request merged**